recipe/patches/0005-Ignore-warnings-in-event.cc-and-logging.cc.patch

```diff
@@ -1,20 +1,20 @@
-From ca5a07d1de602eca1d38b41561a84711e57b7096 Mon Sep 17 00:00:00 2001
+From b8c4c38817bb88a84e3cf8b266bb36bfd9485438 Mon Sep 17 00:00:00 2001
 From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Wed, 11 Nov 2020 01:08:16 +0300
-Subject: [PATCH 11/13] Ignore warnings in event.cc and logging.cc
+Date: Fri, 3 Dec 2021 10:59:10 -0800
+Subject: [PATCH 5/6] Ignore warnings in event.cc and logging.cc
 
 Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Signed-off-by: Gregory Shimansky <gregory.shimansky@intel.com>
+Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
```

Comment:
I agree with @vnlitvinov that it's good to keep the attributions. By preference, I personally also try to keep the original commit dates (e.g. `git commit --author="..." --date="..."`) - not gonna make it a condition here, but it would be nice.

This works well in a rebasing workflow, because then the commit message doesn't even have to be re-entered.